scripts/chroot.sh

```diff
@@ -155,10 +155,9 @@ apt-get install -f
 apt-get -qq -y remove dconf-tools
 
 # Instalation of packages from Generator Webapp
-SCRIPT_URL=https://www.github.com/fossasia/meilix-generator/archive/master.zip
-wget -O $scripts.zip $SCRIPT_URL
-unzip scripts.zip
-SCRIPTS_FOLDER_IN_ZIP="meilix-generator-master/scripts"
+SCRIPT_URL=https://github.com/fossasia/meilix-generator/trunk/scripts
+svn checkout $SCRIPT_URL
```

Comment:
why are you using a VCS here ? for downloading ?

Reply:
Because I want to download at particular script folder only not the whole repo.

Reply:
I think, we can change it do DownGit: https://stackoverflow.com/a/38879691 Please check if this is possible. The purpose of this code is to allow the download as a zip file, so we can change it to almost any location because people know how to create zip files.